Gemfile.lock

```diff
@@ -648,4 +671,4 @@ DEPENDENCIES
   xmpp4r (~> 0.5.6)
 
 BUNDLED WITH
-   1.10.6
+   1.11.2
```

Comment:
Bundler spits out a warning when using an older version. Worst case scenario is that devs need to update their version of bundler to suppress the warning which is probably a good thing anyway.